_security-analytics/sec-analytics-config/log-types.md

```diff
@@ -8,22 +8,44 @@ nav_order: 16
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/c-log-type.png" alt="The Log types landing page." width="85%">
+
+The following table shows the log types that are currently supported by Security Analytics for ingestion, mapping, and monitoring. 
+
+| Category | Log type | Description | 
+| :--- |:--- |:--- |
+| Access Management | `Ad_ldap` | Active Directory logs that track LDAP queries, errors from the LDAP server, timeout events, and unsecure LDAP binds. |
+| Access Management | `Apache_access` | Apache access logs that record data for all requests processed by an Apache HTTP server. |
+| Access Management | `Okta` | Okta logs that record Okta events from a range of actions such as downloading an export file, requesting an application access, or revoking privileges. |
+| Applications | `GitHub` | GitHub logs that monitor workflows created by [GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions). |
+| Applications| `Gworkspace` | Google Workspace logs that monitor log entries such as admin actions, group and group membership actions, and events having to do with logging in. |  
+| Applications| `M365` | Microsoft 365 audit logs that collect a range of data for Microsoft 365 including records from call details, performance data, SQL Server, security events, and access control activity. |
+| Cloud Services | `Azure` | Microsoft Azure logs that monitor log data for cloud applications managed by Microsoft Azure Cloud Services. |
+| Cloud Services | `CloudTrail` | AWS CloudTrail logs that monitor events for an AWS CloudTrail account. OpenSearch can ingest CloudTrail log data from both [AWS Simple Storage Service](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) (S3) accounts and [Amazon Security Lake](https://docs.aws.amazon.com/security-lake/latest/userguide/what-is-security-lake.html) service accounts. | 
+| Cloud Services | `S3` | AWS S3 logs that track requests for access to an Amazon S3 bucket. |
+| Network Activity| `Dns` | DNS logs that store Domain Name System (DNS) activity. |
+| Network Activity | `Network` | Network logs that record events that happen in a system's network, such as login attempts and application events. | 
+| Network Activity | `vpcflow` | [VPC Flow Logs](https://docs.aws.amazon.com/prescriptive-guidance/latest/logging-monitoring-for-application-owners/vpc-flow-logs.html) that capture information about the IP traffic going to and from network interfaces in your VPC. |
+| Security | `Waf` | Web Application Firewall (WAF) logs (introduced in OpenSearch 2.11) for users that require monitoring of the WAF use case that's provided out-of-the box in the Security plugin. The role of WAF is to monitor and filter HTTP traffic between a web application and the internet. WAF prevents common security attacks, such as cross-site scripting (XSS), SQL Injection (SQi), and so on. |
```

Comment:
Are we referring to AWS WAF here? If so, the service name must always be "AWS WAF", never shortened to "WAF", and we don't define WAF as "Web Application Firewall". Instead, we say "AWS WAF is a web application firewall."